x-pack/platform/test/api_integration_deployment_agnostic/default_configs/serverless.config.base.ts

```diff
@@ -110,10 +110,6 @@ export function createServerlessTestConfig<T extends DeploymentAgnosticCommonSer
         ...svlSharedConfig.get('esTestCluster'),
         serverArgs: [
           ...svlSharedConfig.get('esTestCluster.serverArgs'),
-          // custom native roles are enabled only for search and security projects
-          ...(options.serverlessProject !== 'oblt'
-            ? ['xpack.security.authc.native_roles.enabled=true']
```

Comment:
Don't we want to keep the ES 'xpack.security.authc.native_roles.enabled=true' setting, but for all project types? I think we just need to remove the oblt condition.

Reply:
I thought that since it was true already in all the other base config files, it was redundant? I wasn't sure I was just gonna let the tests run and see.

Reply:
You mentioned before
> Nit: as this defaults to true, I think we can just remove it, along with the comment.
So if it defaults to true, why do we need to add it in the test base?